events/api.py

```diff
@@ -520,6 +520,14 @@ class LinkedEventsSerializer(TranslatedModelSerializer, MPTTModelSerializer):
     )
     only_admin_visible_fields = ("created_by", "last_modified_by")
 
+    def are_only_admin_visible_fields_allowed(self, obj):
```

Comment:
Small nitpick; move this method after `__init__`. It's like an unspoken rule, but the constructor should almost always be the first method in a class. Doesn't really affect anything else than readability.